src/hooks/useDisableInputUntilReply.ts

```diff
@@ -17,18 +18,26 @@ export const useDisableInputUntilReply = ({
   lastMessage,
   botUser,
   currentUserId,
+  blockInputWhileBotResponding,
 }: UseDisableInputUntilReplyProps) => {
   const timerRef = useRef<ReturnType<typeof setTimeout> | undefined>(undefined);
-
   const [isMessageInputDisabled, setIsMessageInputDisabled] = useState(false);
 
   useEffect(() => {
-    if (!currentUserId || !botUser || !lastMessage) return;
+    if (
+      !blockInputWhileBotResponding ||
+      !currentUserId ||
+      !botUser ||
+      !lastMessage
+    )
+      return;
     if (isSentBy(lastMessage, currentUserId) && lastMessage.messageId === 0) {
       setIsMessageInputDisabled(true);
-      timerRef.current = setTimeout(() => {
-        setIsMessageInputDisabled(false);
-      }, 10000);
+      if (typeof blockInputWhileBotResponding === 'number') {
```

Comment:
If it is used as true, what is the default duration in seconds?

Reply:
Also, it seems there is no logic that executes when it is used as true.

Reply:
I added note to the exported interface
```
export interface MessageInputControls {
  /**
   * If undefined, false, or 0 is given, message input will not be blocked.
   * If true or non-zero numeric value is given, message input will be blocked.
   * If true is given, there will not be a force unblock.
   * If numeric value is given, it will be used as timeout to force unblock the input.
   */
  blockWhileBotResponding?: boolean | number;
}
```
> If true is given, there will not be a force unblock.

I wanted to give them option to turn off force unblock

Reply:
Oh, I didn't see that. Thank you for letting me know.